internal/store/certificatesigningrequest_test.go

```diff
@@ -27,18 +27,6 @@ import (
 )
 
 func TestCsrStore(t *testing.T) {
-	const metadata = `
```

Comment:
Why are we removing these? We do want these in output that's checked no?

Reply:
I am afraid we do not need `metadata` for now. I have not checked out the details but golangci-tools reports that `metadata` is not referenced anymore.

Reply:
This seems to be a leftover from a refactor that was done on the collectors package. https://github.com/kubernetes/kube-state-metrics/blob/release-1.4/pkg/collectors/daemonset_test.go#L120

Reply:
But wouldn't that mean that instead of removing it we want to prepend it to all "want" cases in the tests?

Reply:
@brancz It wasn't just the input, but the entire way the testing happens that was changed. The util testing tool for collectors changed. Before the [GatherAndCompare](https://github.com/kubernetes/kube-state-metrics/blob/release-1.4/pkg/collectors/testutils/testutils.go#L34 ) function was used which is removed now. That used the prometheus registry helper functions for gathering and comparing the metrics. Instead, the [new testutils](https://github.com/kubernetes/kube-state-metrics/blob/f24d831a4743b80f294193943b3f91a8b8308286//internal/store/testutils.go#L39:42) helpers are used now.

But I agree the tests should be testing the presence of the metadata! I just don't think that's necessarily in the scope of this PR.

Reply:
Yes, however, I'd prefer to use the `metadata` variable to resolve the linting issues, instead of removing it.

Reply:
Maybe we should first merge this and then another PR needs to adjust the testing utils to leverage the `metadata`.  1.7.0 should be cut after the `metadata` refactor is done.

Reply:
I'm not suggesting the testutils to actually make use of it now. Just use the `metadata` variable like it is in other tests already. I'd prefer to keep this around and then fix things as needed when we actually use it in the testutils again.

Reply:
Sorry I'm not sure I was clear enough in what I meant. I meant use `metadata` in the `expected` just so we don't loose it, that way when we adapt the testutils to actually test metadata again we will have less to get back and or see missing or wrong metadata tests immediately.

Reply:
@andyxning if you rebase your PR the above problem of removing metadata won't be needed anymore it was fixed in https://github.com/kubernetes/kube-state-metrics/pull/848